### DISCUSSION TITLE
Welcome to TastyEstate Discussions!

### DISCUSSION THREAD

**Discussion by junhong91** [matgip/TastyEstate#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by junhong91** [comment]
Vue script영역 ground rule 정하는게 좋을듯

**Reply by junhong91** [reply]
component >  mounted > computed > method 순?

**Reply by jkpark** [reply]
상단 : component, data, props 등 변수선언 관련
중단 :  computed(), watch(), craeted(), mounted(), unmounted() 등 정의된 메소드들
하단 : method()

**Reply by jkpark** [reply]
모든 스타일링은 Vue 공식 가이드를 따름
https://vuejs.org/style-guide/

**Comment by junhong91** [comment]
지금 components/Reviews 디렉토리 내부에서  Review/ReviewDiag 디렉토리가 있는데 굳이 이렇게 하위로 나눠 놓을 필요가 있을까?? Flat하게 합치는게 더 가독성이 높아질거같은데
